pq/models.py

```diff
@@ -274,6 +274,16 @@ def queue_task(self, jobname, meta_params=None, **kw):
         meta_params['from_task'] = self.task.id
         return self.backend.queue_task(jobname, meta_params=meta_params, **kw)
 
+    def queue_task_local(self, jobname, meta_params=None, **kw):
+        '''Queue a new task in the local task queue
+        '''
+        assert self.backend, 'backend not available'
+        if meta_params is None:
+            meta_params = {}
+        meta_params['from_task'] = self.task.id
```

Comment:
Makes sense.
Can we use one function for both ``queue_task`` and ``queue_task_local``, something like this?
```python
meta_params = self._meta_params(meta_params)
```